pyproject.toml

```diff
@@ -4,7 +4,7 @@ version = "0.0.1"
 description = "End-to-end aframe pipeline components"
 authors = [
     "Alec Gunny <alec.gunny@ligo.org>",
-    "Ethan Marx",
+    "Ethan Marx <emarx@mit.edu>",
     "William Benoit"
```

Comment:
Would you mind adding my email here as well? benoi090@umn.edu. And we should probably move Alec to the bottom so that if anyone reading this wants to email someone, they don't start with him.